codice/ddf
# DDF-04538: Refactored 'time-settings' to React; removed old code

#### What does this PR do?
Refactors `time-settings` to React
#### Who is reviewing it? 
@Bdthomson 
@jrnorth 
@brianfelix 
@Corey-Collins 
#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@bdeining

#### How should this be tested?
1. Open the settings 'gear' icon and select `Time`
2. Verify the time is ticking
3. Select different time zone and date format settings
4. Reload the application to verify selected settings have been persisted
#### Any background context you want to provide?
#### What are the relevant tickets?
For GH Issues:
Fixes: #4538 

#### Screenshots
<img width="1792" alt="Screen Shot 2019-03-13 at 9 34 13 AM" src="https://user-images.githubusercontent.com/118054/54283727-ffb3b100-4574-11e9-8eac-2d426c262899.png">
<img width="1792" alt="Screen Shot 2019-03-13 at 9 34 19 AM" src="https://user-images.githubusercontent.com/118054/54283730-004c4780-4575-11e9-9a40-9f00a0c16466.png">
<img width="1792" alt="Screen Shot 2019-03-13 at 9 34 27 AM" src="https://user-images.githubusercontent.com/118054/54283731-00e4de00-4575-11e9-9328-25d423236f4f.png">

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
